raiden/routing.py

```diff
@@ -213,6 +213,8 @@ def get_best_routes_internal(
         # https://github.com/raiden-network/raiden/issues/4751
         # Internal routing doesn't know about fee so it should set a percentage per hop
         estimated_fee = FeeAmount(round(INTERNAL_ROUTING_DEFAULT_FEE_PERC * amount))
+        if neighbour.length == 0:  # Target is our direct neighbour, pay no fees.
```

Comment:
Are you sure this is the right check? I would have expected `length == 1`.

Reply:
yea i mistakenly set it to 0... testing showed me i used the wrong value.